tests/rkt_net_test.go

```diff
@@ -338,6 +338,80 @@ func TestNetDefaultRestrictedConnectivity(t *testing.T) {
 	f("--net=default-restricted")
 }
 
+/*
+ * Default net port forwarding connectivity
+ * ---
+ * Container launches http server on all its interfaces
+ * Host must be able to connect to container's http server on it's own interfaces
```

Comment:
should I just clean up the whole file comment-wise (separate PR)? I wasn't familiar with Go style guidelines and thought that was a good idea.